### PR TITLE
Add more HCI command and event packets

### PR DESCRIPTION
**Pull request by gsingh93**
**Checklist:**

-   [X] If you are new to Scapy: I have checked [CONTRIBUTING.md](https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md) (esp. section submitting-pull-requests)
-   [X] I squashed commits belonging together
-   [x] I added unit tests or explained why they are not relevant
-   [x] I executed the regression tests (using `cd test && ./run_tests` or `tox`)
-   [X] If the PR is still not finished, please create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)

This PR adds some additional HCI command and event types. With these additional commands and events, it is possible to create a full Bluetooth Classic connection with an endpoint. For reference, these are section numbers in Volume 4, Part E of the Bluetooth 5.3 specification where these types are defined: 7.1.5, 7.1.10, 7.1.15, 7.1.16, 7.1.19

### CODE REVIEW

**Comment by gsingh93** [comment]
This is ready for review, can someone kick off the required workflows?

I'm not sure why test coverage decreased 2.6%, my added code should have 100% coverage: https://app.codecov.io/gh/secdev/scapy/pull/4003

**Comment by gsingh93** [comment]
Ping @guedou or @p-l-, can I get a review for this?

**Review by p-l- — APPROVED** [review]

File: test/scapy/layers/bluetooth.uts [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
 = HCI layers
 # a huge packet with all classes in it!
-pkt = HCI_ACL_Hdr()/HCI_Cmd_Complete_Read_BD_Addr()/HCI_Cmd_Connect_Accept_Timeout()/HCI_Cmd_Disconnect()/HCI_Cmd_LE_Connection_Update()/HCI_Cmd_LE_Create_Connection()/HCI_Cmd_LE_Create_Connection_Cancel()/HCI_Cmd_LE_Host_Supported()/HCI_Cmd_LE_Long_Term_Key_Request_Negative_Reply()/HCI_Cmd_LE_Long_Term_Key_Request_Reply()/HCI_Cmd_LE_Read_Buffer_Size()/HCI_Cmd_LE_Set_Advertise_Enable()/HCI_Cmd_LE_Set_Advertising_Data()/HCI_Cmd_LE_Set_Advertising_Parameters()/HCI_Cmd_LE_Set_Random_Address()/HCI_Cmd_LE_Set_Scan_Enable()/HCI_Cmd_LE_Set_Scan_Parameters()/HCI_Cmd_LE_Start_Encryption_Request()/HCI_Cmd_Read_BD_Addr()/HCI_Cmd_Reset()/HCI_Cmd_Set_Event_Filter()/HCI_Cmd_Set_Event_Mask()/HCI_Command_Hdr()/HCI_Event_Command_Complete()/HCI_Event_Command_Status()/HCI_Event_Disconnection_Complete()/HCI_Event_Encryption_Change()/HCI_Event_Hdr()/HCI_Event_LE_Meta()/HCI_Event_Number_Of_Completed_Packets()/HCI_Hdr()/HCI_LE_Meta_Advertising_Reports()/HCI_LE_Meta_Connection_Complete()/HCI_LE_Meta_Connection_Update_Complete()/HCI_LE_Meta_Long_Term_Key_Request()
```
Comment (gpotter2) [inline test/scapy/layers/bluetooth.uts:7]: Sorry for the delay. Could you add a proper test? (build/dissect some packets). This one is absurd and should eventually be deleted.
Thanks !

**Review by gpotter2 — CHANGES_REQUESTED** [review]

**Comment by gsingh93** [comment]
@gpotter2 I added some more tests, can you take another look?

**Comment by p-l-** [comment]
Hey @gsingh93 can you rebase against current master? That should fix the failing test.

**Comment by gsingh93** [comment]
@p-l- done

**Comment by p-l-** [comment]
@gsingh93 perfect. Now @gpotter2 is it OK on your side (since you had requested changes)?

**Review by gpotter2 — APPROVED** [review]
Thanks for addressing this !

**Comment by p-l-** [comment]
Thank you @gsingh93!

**Pull request merged**